### PR TITLE
[FIX][account_constraints] Fix account_constraints to works with account_followup

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Currently, account_constraints doesn't work with account_followup

### CODE REVIEW

File: account_constraints/model/account_move_line.py [hunk 1]
```diff
@@ -28,7 +28,8 @@ class AccountMoveLine(models.Model):
     def _get_write_authorized_fields(self):
         """ This method can be overrride to add some field that can be written
         directly on account move line """
-        return ["reconcile_id", "reconcile_partial_id"]
+        return ["reconcile_id", "reconcile_partial_id", "followup_line_id",
```
Comment (pedrobaeza) [inline account_constraints/model/account_move_line.py:31]: Are these fields included in the account module or we should depend now on account_followup?

Comment (sbidoul) [inline account_constraints/model/account_move_line.py:31]: @pedrobaeza these fields are part of the official account_followup.

The most correct solution would be an auto_install module that glues account_followup and account_constraints.

But I'm not shocked with the solution proposed by @adrienpeiffer

**Comment by sbidoul** [comment]
:+1:

**Comment by pedrobaeza** [comment]
OK, I see there's no problem with adding non-existing fields to the method, so :+1:

**Pull request merged**